configs/pontoon.toml

```diff
@@ -189,6 +189,52 @@ locales = [
 [[paths]]
     reference = "en/mozorg/about/history.ftl"
     l10n = "{locale}/mozorg/about/history.ftl"
+[[paths]]
+    reference = "en/mozorg/about/legal.ftl"
+    l10n = "{locale}/mozorg/about/legal.ftl"
+    locales = [
```

Comment:
Should this be open to all locales rather than those with partial/full localization?